lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1877,7 +1872,7 @@ LLVMToSPIRVBase::transValueWithoutDecoration(Value *V, SPIRVBasicBlock *BB,
     if (!BM->getErrorLog().checkError(
             !AtomicRMWInst::isFPOperation(Op) && Op != AtomicRMWInst::Nand,
             SPIRVEC_InvalidInstruction,
-            OCLUtil::toString(V) + "\nAtomic " +
+            toString(V) + "\nAtomic " +
```

Comment:
```suggestion
            V + "\nAtomic " +
```

Reply:
Do you mean such change? It'll not work now as `checkError` accepts an `Instruction`, not `Value`.
```suggestion
            V, "\nAtomic " +
```

Reply:
Should we change `checkError` to accept `Value`?

Reply:
Applied in ff6cabe.